server/forge/github/convert.go

```diff
@@ -168,3 +178,15 @@ func convertLabels(from []*github.Label) []string {
 	}
 	return labels
 }
+
+func convertCommit(from *github.Commit) *model.Commit {
+	return &model.Commit{
+		SHA:      from.GetSHA(),
+		ForgeURL: from.GetHTMLURL(),
+		Message:  from.GetMessage(),
+		Author: model.CommitAuthor{
+			Author: from.GetAuthor().GetName(),
```

Comment:
This is actually a quite nice idea to use `author.name` instead of `author.author`